oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -508,41 +507,28 @@ private List<String> extractVoTStringListFromIdTokenHint(SignedJWT idTokenHint)
     }
 
     private APIGatewayProxyResponseEvent handleDocAppJourney(
-            Optional<String> existingSessionId,
             Optional<OrchSessionItem> orchSessionOptional,
             OrchClientSessionItem orchClientSession,
             AuthenticationRequest authenticationRequest,
             ClientRegistry client,
             String clientSessionId,
             String persistentSessionId,
             TxmaAuditUser user) {
-        Session session;
         var newSessionId = IdGenerator.generate();
         var newBrowserSessionId = IdGenerator.generate();
-        var existingSession = existingSessionId.flatMap(sessionService::getSession);
-        if (existingSessionId.isEmpty() || existingSession.isEmpty()) {
-            session = sessionService.generateSession();
-            updateAttachedSessionIdToLogs(newSessionId);
-            LOG.info("Created new session with ID {}", newSessionId);
-        } else {
-            var previousSessionId = existingSessionId.get();
-            session =
-                    sessionService.updateWithNewSessionId(
-                            existingSession.get(), previousSessionId, newSessionId);
-            updateAttachedSessionIdToLogs(newSessionId);
-            LOG.info("Updated session ID from {} to {}", previousSessionId, newSessionId);
-        }
```

Comment:
Since this is changing more than just a `get`, presumably this PR can't be merged until Auth stop reading this session entirely?